generate_queries_openai.py

```diff
@@ -133,8 +126,8 @@ def parse_bad_good(data):
                 if line_num & (line_num - 1) == 0 or line_num % 1000 == 0:
                     # LOG every power of 2 or 1000 steps.
                     print(f'Document: {doc_text}\nQuestion: {question}\n')
+                
                 num_examples_so_far += 1
                 progress_bar.update(1)
 
-                time.sleep(args.sleep_time)
```

Comment:
Isn't this needed to avoid rate limiting?

Reply:
sleep was added again